model/service.go

```diff
@@ -107,6 +112,19 @@ const (
 	ProtocolRedis Protocol = "Redis"
 )
 
+type SecurityOption string
+
+const (
+	// Enable authentication. This overrules mesh's AuthPolicy.
+	SecurityEnable SecurityOption = "enable"
+
+	// Disable authentication. This overrules mesh's AuthPolicy.
+	SecurityDisable SecurityOption = "disable"
+
+	// Set authentication as specified by mesh's AuthPolicy.
+	SecurityDefault SecurityOption = ""
+)
+
```

Comment:
This should move into platform/kube .. These annotations are platform specific. Its responsibility of platform adapter to enable/disable the boolean under the port. After that, you shouldn't have to worry.

Reply:
As mentioned above, boolean is too limit. This is more or less an enum of all options.